Bottom: 93993b952dd7409cf7520a2df27a564d07c761a4
Top:    ed4516d4ed4181f231acbe1a75694580487d7162
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-04 11:45:07 +0800

Refresh of vhost-packed-ring-support

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 90d9efb..994711b 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -74,8 +74,7 @@ enum {
 	VHOST_NET_FEATURES = VHOST_FEATURES |
 			 (1ULL << VHOST_NET_F_VIRTIO_NET_HDR) |
 			 (1ULL << VIRTIO_NET_F_MRG_RXBUF) |
-			 (1ULL << VIRTIO_F_IOMMU_PLATFORM) |
-			 (1ULL << VIRTIO_F_RING_PACKED)
+			 (1ULL << VIRTIO_F_IOMMU_PLATFORM)
 };
 
 enum {
